### DISCUSSION TITLE
How to ensure standalone windows exe is shut down correctly?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [nocodb/nocodb#9546] [General]
If I log off, close the tab, and then press ctrl-D and enter or close the command window via the windows GUI, the window closes instantly. But I'm not sure if this is actually nocodb shutdown and clean up properly. How can I ensure I am shutting it down correctly?

**Comment by Vetri1706** [comment]
Check whether the programs background process is running in the task manager, just don't force it to shut down else your system may even crash.

**Reply by mariomadproductions** [reply]
Yeah basically I'm just not sure if the shut down procedure I described is actually a safe way to shut down the software.
